src/components/usercomponents/NotificaitonComponent/NotificationFilters.tsx

```diff
@@ -0,0 +1,34 @@
+interface NotificationFiltersProps {
+  filter: 'all' | 'unread' | 'today';
+  setFilter: (filter: 'all' | 'unread' | 'today') => void;
+}
+
+const NotificationFilters: React.FC<NotificationFiltersProps> = ({
+  filter,
+  setFilter,
+}) => {
+  return (
+    <div className="mb-2 flex justify-between">
+      <button
+        className={`text-sm ${filter === 'all' ? 'font-semibold' : 'text-gray-600'}`}
+        onClick={() => setFilter('all')}
+      >
+        All
+      </button>
+      <button
+        className={`text-sm ${filter === 'unread' ? 'font-semibold' : 'text-gray-600'}`}
+        onClick={() => setFilter('unread')}
+      >
+        Unread
+      </button>
+      <button
+        className={`text-sm ${filter === 'today' ? 'font-semibold' : 'text-gray-600'}`}
+        onClick={() => setFilter('today')}
+      >
+        Today
+      </button>
```

Comment:
You just created a new button component in the notification component directory, why not use it?

Reply:
??